build.html

```diff
@@ -8,7 +8,7 @@
   {{{advancedSettings.cssCode}}}
   </style>
   {{/if}}
-  <div data-dynamic-lists-id="{{ id }}" data-dynamic-lists-layout="{{ layout }}" data-dynamic-lists-uuid="{{uuid}}"></div>
+  <div data-dynamic-lists-id="{{ id }}" data-dynamic-lists-layout="{{ layout }}" data-dynamic-lists-uuid="{{uuid}}" class="dynamic-list-layout"></div>
```

Comment:
What do we still need the `.dynamic-list-layout` class for?

```suggestion
  <div data-dynamic-lists-id="{{ id }}" data-dynamic-lists-layout="{{ layout }}" data-dynamic-lists-uuid="{{uuid}}"></div>
```

Reply:
@tonytlwu I'm sorry, I forgot to remove this class from html.